test/gateway/gateway_base/bounty_change.js

```diff
@@ -0,0 +1,156 @@
+const GatewayBase = artifacts.require("./GatewayBase.sol")
+  , BN = require('bn.js');
+
+const Utils = require('../../../test/lib/utils');
+
+async function proposeBountyChange(gatewayBaseInstance, proposedBounty, organisation, currentBounty) {
+
+  let actualProposedBounty = await gatewayBaseInstance.initiateBountyAmountChange.call(proposedBounty, {from: organisation});
+
+  assert(proposedBounty.eq(actualProposedBounty));
+
+  let response = await gatewayBaseInstance.initiateBountyAmountChange(proposedBounty, {from: organisation});
+
+  let expectedUnlockHeight = response.receipt.blockNumber + 100;
+
+  let expectedEvent = {
+    BountyChangeInitiated: {
+      _currentBounty: currentBounty,
+      _proposedBounty: proposedBounty,
+      _unlockHeight: new BN(expectedUnlockHeight)
+    }
+  };
+
+  assert.equal(
+    response.receipt.status,
+    1,
+    "Receipt status is unsuccessful"
+  );
+
+  let eventData = response.logs;
+  await Utils.validateEvents(eventData, expectedEvent);
+  return expectedUnlockHeight;
+}
+
+contract('GatewayBase.sol', function (accounts) {
+
+  describe('Initiate change bounty', async () => {
+
+    let gatewayBaseInstance;
+    let organisation = accounts[2];
+
+    beforeEach(async function () {
+
+      let core = accounts[0]
+        , messageBus = accounts[1]
+        , bounty = new BN(100);
+
+      gatewayBaseInstance = await GatewayBase.new(
+        core,
+        messageBus,
+        bounty,
+        organisation
+      );
+
+    });
+
+    it('should be able to propose bounty change', async function () {
+
+      let proposedBounty = new BN(50);
+      let currentBounty = new BN(100);
+
+      await proposeBountyChange(gatewayBaseInstance, proposedBounty, organisation, currentBounty);
+
+    });
+
+    it('should be able to re-propose bounty change', async function () {
+
+      let proposedBounty = new BN(50);
+      let currentBounty = new BN(100);
+
+      await proposeBountyChange(gatewayBaseInstance, proposedBounty, organisation, currentBounty);
+
+       proposedBounty = new BN(150);
+       currentBounty = new BN(100);
+
+      await proposeBountyChange(gatewayBaseInstance, proposedBounty, organisation, currentBounty);
+    });
+
+    it('only organization should be able to propose bounty change', async function () {
```

Comment:
ideally, these should generally read like a sentence in JS tests. Starting with the `it`. Like `it should not accept a bounty change from another address than the organization` or something like that.